src/genie/libs/parser/iosxe/show_cts_pacs.py

```diff
@@ -0,0 +1,130 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ==================
+# Schema for:
+#  * 'show cts pacs'
+# ==================
+class ShowCtsPacsSchema(MetaParser):
+    """Schema for show cts pacs."""
+
+    schema = {
+        "cts_pacs_info": {
+            "aid": str,
+            "pac_type": str,
+            "iid": str,
+            "aid_info": str,
+            "lifetime": {
+                "date": str,
+                "time": str,
+                "time_zone": str
+            },
+            "pac_opaque": str,
+            "refresh_timer": str
+        }
+    }
+
+
+# ==================
+# Parser for:
+#  * 'show cts pacs'
+# ==================
+class ShowCtsPacs(ShowCtsPacsSchema):
+    """Parser for show cts pacs"""
+
+    cli_command = ['show cts pacs']
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command[0])
+        else:
+            out = output
+
+
+        cts_pacs_dict = {}
+
+        aid_capture = re.compile(r"^AID:\s+(?P<aid>\S+)", re.MULTILINE)
```

Comment:
Could you please add some sample output on top as comment. This will help us check the regex if any issue.
```python
#AID: 1100E046659D4275B644BF946EFA49CD
aid_capture = re.compile(r"^AID:\s+(?P<aid>\S+)", re.MULTILINE)

Reply:
I think you can remove `re.MULTILINE`, as you splited the output into one line in `def filter_lines(raw_output, remove_lines)`

```
aid_capture = re.compile(r"^AID:\s+(?P<aid>\S+)")

Reply:
@LenzDong, Removed multiline, and provided the sample output next to the regex patterns.